tools/flaky-test-reporter/error.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// error.go helps with error handling
+
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+// combineErrors combines slice of errors and return a single error
+func combineErrors(errs []error) error {
+	if nil == errs || 0 == len(errs) {
```

Comment:
not needed. The loop below will handle empty slices just fine. Only things night be we will be adding a "\n" to the list if its empty, which should be fine.

Reply:
removed